bluepymm/prepare_combos/prepare_emodel_dirs.py

```diff
@@ -300,10 +300,9 @@ def prepare_emodel_dirs(
              continu))
 
     print('Parallelising preparation of e-model directories')
-    pool = multiprocessing.Pool(maxtasksperchild=1)
+    pool = multiprocessing.Pool(processes=1, maxtasksperchild=1)
```

Comment:
Is this basically disabling the multiprocessing?

Reply:
yes, if I replaces pool.map by map, it didn't work, I don't know why.

Reply:
And why does it have to be disabled?

Reply:
for some obscure reason, it is really slow with it. it starts many cc1 processes, I don't know why. We can keep it as it was, I'll just disable for my tests.